### DISCUSSION TITLE
add 'unit' export in csv_export and not just column names

### DISCUSSION THREAD

**Discussion by AlexanderKrabbe** [ixdat/ixdat#53] [Ideas]
In the ixdat_csv reader there is a lookup from regular expression to figure out if a column_name has a unit in it e.i 'M44 / [A]' -> Unit('A') 

However in the export module csv_exporter (which I presume is the counterpart to ixdat_csv and might be wrong of this) does not export the unit but only the column names.

Since cinfdata "expects" all columns that start with M to be signals and gives 'A' as default unit. This is then not reflected in the columns name and the unit gets lost in the csv_exporter. 

```python
""" First read cinfdata.txt with cinfreader """
meas = Measurement.read('baratron_temp_measurement.txt', reader='cinfdata')
meas['M2'].unit
Unit('A')

""" Export with csv """
meas.export('cinfdata')

""" second read of cinfdata now as .csv  / shared data to external computer to open data"""
meas = Measurement.read('cinfdata.csv', reader='ixdat')
meas['M2'].unit
Unit('')

```

This is an issue primarily when data is retrieved from a pdatabase like cinf_database and sequentially exported to share data with collaborators.

Maybe the ixdat_csv reader could have an extra argument with regular_expressions for user to set unit_name to known columns without the unit being part of the column name?

Or maybe units just needs to be manually added after import of data.

**Comment by KennethNielsen** [comment]
@AlexanderKrabbe interesting point. I would assume that the exporter should try and export the units when they are there. So if that would solve the problem, we shoud make a feature request issue that covers it.

**Reply by AlexanderKrabbe** [reply]
That might solve this and be a nice feature. 

Another solution could be the csv_reader to guess on units for the masses like cinftxtreader does?
